src/Microsoft.TestPlatform.Client/RequestHelper/ITestRequestManager.cs

```diff
@@ -88,6 +88,18 @@ public interface ITestRequestManager : IDisposable
         ITestSessionEventsHandler eventsHandler,
         ProtocolConfig protocolConfig);
 
+    /// <summary>
+    /// Stops a test session.
+    /// </summary>
+    /// 
+    /// <param name="testSessionInfo">The test session info.</param>
+    /// <param name="eventsHandler">The events handler.</param>
+    /// <param name="protocolConfig">Protocol related information.</param>
+    public void StopTestSession(
```

Comment:
I don't think you wanted to include accessibility.

```suggestion
    void StopTestSession(
```